lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -174,11 +174,19 @@ def failure_slash_error_lines
           lines
         end
 
+        # rubocop:disable Lint/RescueException
+        def exception_message_string(exception)
+          exception.message.to_s
+        rescue Exception => other
```

Comment:
```
NoMemoryError.superclass # => Exception
```
maybe rescue from `StandardError` here?

Reply:
No the point is to avoid crashing due to calling this line.

Reply:
I mean we can't recover from a `NoMemoryError` by swallowing it.

On the other hand, `SystemStackError` is also an `Exception`, and it may be caused by an incorrectly implemented `message`. 👍

Reply:
Well we might be able to because we're not necessarily assigning memory, but Ruby doesn't have a system level of exception unfortunately